docs/migration-guide/to-16.md

```diff
@@ -0,0 +1,13 @@
+# Migrating to 16.0.0
+
+This release contains breaking changes.
+
+## Breaking changes
+
+This breaking change may affect you:
+
+- removed deprecated stylistic rules
```

Comment:
```suggestion
```

We can add the index when we have more than one change. Let's keep it simple for now.